multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/template/TemplateFactory.java

```diff
@@ -88,9 +121,153 @@ public final void fillTemplateAuth(final String filePathToSave, final String aut
 
   }
 
+  public final void fillTemplateNotNull(final String filePathToSave) throws IOException, TemplateException {
```

Comment:
The same coment as in async version; and I don't know if it would be possible to unify both of them (and yes, I have seen that async methods have two parameters).

Reply:
The same fix you suggested for asyncApi has been done, but I think unifying both files would be a big refactor and is maybe better done in a new separate issue